homeassistant/components/homeassistant_hardware/strings.json

```diff
@@ -73,7 +73,8 @@
         "otbr_still_using_stick": "This {model} is in use by the OpenThread Border Router add-on. If you use the Thread network, make sure you have alternative border routers. Uninstall the add-on and try again.",
         "unsupported_firmware": "The radio firmware on your {model} could not be determined. Make sure that no other integration or add-on is currently trying to communicate with the device. If you are running Home Assistant OS in a virtual machine or in Docker, please make sure that permissions are set correctly for the device.",
         "fw_download_failed": "{firmware_name} firmware for your {model} failed to download. Make sure Home Assistant has internet access and try again.",
-        "fw_install_failed": "{firmware_name} firmware failed to install, check Home Assistant logs for more information."
+        "fw_install_failed": "{firmware_name} firmware failed to install, check Home Assistant logs for more information.",
+        "not_user_configurable": "This integration is not user configurable. Please plug in supported Home Assistant hardware and configure it using the discovered device."
```

Comment:
Note: The actual `not_user_configurable` string for the base `homeassistant_hardware` integration is located under `firmware_picker` -> `options` right now.
Technically, this is a `config` flow, not an `options` flow, so do we want to move this string under a new `config` entry here (just for that string)?

The other hardware integrations also refer to the `options` strings here from their `config` section/strings, but this user step (and abort reason) can never happen in the `options` flow, so do we want to move this? (I don't *really* see a need, but it would be more correct probably.)